test/strategy/rysk/RyskStrategy.spec.ts

```diff
@@ -338,6 +338,22 @@ describe('RyskStrategy', () => {
         strategy.connect(manager).withdrawToVault(endAmountToWithdraw),
       ).to.be.revertedWith('RyskPendingWithdrawalNotCompleted');
     });
+
+    it('works when called after completing previous withdrawal in the same epoch', async () => {
+      let underlyingAmount = parseUSDC('100');
```

Comment:
why let?

Reply:
not intentional, [fixed](https://github.com/lindy-labs/sc_solidity-contracts/pull/216/commits/fec0fc34a03f3e28b4cf8ed548b8720d607c61ec)